day00/ex02/ex02.md

```diff
@@ -0,0 +1,65 @@
+# Exercise 02 - Variance
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turn-in directory :   |  ex02              |
+|   Files to turn in :    |  variance.py       |
+|   Forbidden functions : |  *.sum(), np.mean(), np.var() |
+|   Remarks :             |  n/a               |
+
+## Objectives:
+
+You must implement the following formula as a function:  
```

Comment:
maybe explain what is variance, why it is used (gaussian curve, std) ? or ask for a question ?